src/components/secondPassphraseInput/index.test.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import chai, { expect } from 'chai';
+import sinonChai from 'sinon-chai';
+import { mount } from 'enzyme';
+import { Provider } from 'react-redux';
+import SecondPassphraseInputContainer from './';
+import store from '../../store';
+
+chai.use(sinonChai);
+
+
```

Comment:
If you think we shouldn't use more than one line, please open an Issue to update eslint http://eslint.org/docs/rules/no-multiple-empty-lines